lessons/micropython/mini-workshop/index.md

```diff
@@ -0,0 +1,319 @@
+
+!!! note ""
+
+    Tento workshop nejde projít jen tak z domu – je potřeba speciální příprava.
+    Pokyny pro organizátory najdeš
+    na [podstránce]({{ subpage_url('organizers') }}).
+
+Jedna varianta Pythonu, MicroPython, běží na malých zařízeních.
+Taková zařízení si teď můžeš vyzkoušet.
+
+Jak na to?
+
+
+## Otevření příkazové řádky
+
+<img src="{{ static('icon_terminal.png')}}" style="float:right;height:5em;">
+
+Klikni na *Activities*; potom vyber z levého panelu *Terminal* (ikonka
+černého okýnka).
+
+Objeví se černé okýnko s bílým textem, kam můžeš zadávat příkazy – příkazová
+řádka.
+
+
+
+## Otevření komunikačního kanálu
+
+V příkazové řádce by měl být řádek končící dolarem.
+Za ten dolar napiš:
+
+```bash
+picocom -b 115200 /dev/ttyUSB0
+```
+
+Opisuj opatrně, každé písmenko má smysl!
+Pak zmáčkni Enter.
+Měl by se objevit následující výpis:
+
+```plain
+picocom v1.7
+
+port is        : /dev/ttyUSB0
+flowcontrol    : none
+baudrate is    : 115200
+parity is      : none
+databits are   : 8
+escape is      : C-a
+local echo is  : no
+noinit is      : no
+noreset is     : no
+nolock is      : no
+send_cmd is    : sz -vv
+receive_cmd is : rz -vv
+imap is        : 
+omap is        : 
+emap is        : crcrlf,delbs,
+
+Terminal ready
+```
+
+Pak stiskni Enter.
+Měly by se objevit tři zobáčky, kterými MicroPython prosí o instrukce:
+
+```plain
+>>>
+```
+
+
+<div style="page-break-after: always;"></div>
+
+## Kalkulačka
+
+Za tři zobáčky můžeš zadat nějaký matematický příklad. MicroPython ho spočítá.
+Zkus třeba tyhle (zobáčky nepiš, ty vypíše sám MicroPython:
+
+```pycon
+>>> 1 + 1
+
+>>> 3 * 3
+
+>>> 1 / 4
+
+>>> (84 + 5) * 253
+```
+
+
+## Světýlko
+
+MicroPython na nach destičkách má připravený objekt `led`, jehož metodou
```

Comment:
Ano, to zařizuje [náš boot.py](https://github.com/pyvec/naucse.python.cz/pull/84/files#diff-059c42f1059520a466c0074138c1af1f)